src/format.rs

```diff
@@ -22,7 +22,7 @@ fn prefix_token(level: &Level) -> String
     format!("{}{}{}", "[".blue().bold(), color::level_color(level, level_token(level)), "]".blue().bold())
 }
 
-pub fn format(buf: &mut Formatter, record: &Record) -> Result<(), Error>
+pub fn format(buf: &mut Formatter, record: &Record<'_>) -> Result<(), Error>
```

Comment:
What's the reason for an anonymous lifetime type parameter here? Is it related to the edition upgrade?

Reply:
Yes -- Rust 2018's set of idioms is such that lifetime should always be indicated now, even if they can be elided. It's preferred to use the elided lifetime rather than omitting the lifetime arg entirely now.

This specifically was introduced when I ran `cargo fix --edition-idioms`, though I don't recall if this repo warns on `rust-2018-idioms` off the top of my head. You can make this a warning, though, with something like [this Rust playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=41cb0d2a2801620c0ac139759332bc55).

Reply:
Alrighty then - that's a really good explanation. I'm happy to go ahead and merge this now. Thanks again for this improvement! :)